net-analyzer/speedtest-cli/speedtest_cli-2.0.2.recipe

```diff
@@ -0,0 +1,84 @@
+SUMMARY="A CLI for testing internet bandwidth using speedtest.net"
+DESCRIPTION="speedtest-cli is a command-line interface for testing internet \
+bandwidth using speedtest.net. Note that latency reported by this tool should \
+not be relied on as a value indicative of ICMP style latency; rather, it is a \
+relative value used for determining the lowest latency server for performing \
+the actual speed test against.
+Also note that its results may also be inconsistent with speedtest.net due to \
+the website's migration to using pure socket tests instead of HTTP based ones, \
+the application being written in Python, \
+differing performance between Python versions, \
+and CPU and Memory capacity and speed."
+HOMEPAGE="https://github.com/sivel/speedtest-cli"
+COPYRIGHT="2012-2018 Matt Martz"
+LICENSE="Apache v2"
+REVISION="1"
+SOURCE_URI="$HOMEPAGE/archive/v$portVersion.tar.gz"
+CHECKSUM_SHA256="5e22f2dcce1c9020f33faf559b22727483f022008a2912b85d573e578374b6a0"
+SOURCE_FILENAME="speedtest-cli-$portVersion.tar.gz"
+SOURCE_DIR="speedtest-cli-$portVersion"
+
+ARCHITECTURES="any"
+
+PROVIDES="
+	$portName = $portVersion
+	"
+REQUIRES="
+	haiku
+	"
+BUILD_REQUIRES="
+	haiku_devel
+	"
+
+PYTHON_PACKAGES=(python python3)
+PYTHON_VERSIONS=(2.7 3.6)
+for i in "${!PYTHON_PACKAGES[@]}"; do
+pythonPackage=${PYTHON_PACKAGES[i]}
+pythonVersions=${PYTHON_VERSIONS[$i]}
+eval "PROVIDES_${pythonPackage}=\"\
+	${portName}_$pythonPackage = $portVersion\
+	\"; \
+REQUIRES_$pythonPackage=\"\
+	haiku\n\
+	cmd:python$pythonVersion\
+	\""
+BUILD_REQUIRES="$BUILD_REQUIRES
+	setuptools_$pythonPackage"
+BUILD_PREREQUIRES="$BUILD_PREREQUIRES
+	cmd:python$pythonVersion"
+done
+
+PROVIDES_python="$PROVIDES_python
+	cmd:speedtest_cli_python
+	"
+REPLACES_python="
+	python_speedtest_cli
+	"
+PROVIDES_python3="$PROVIDES_python3
+	cmd:speedtest_cli_python3
+	"
+REPLACES_python3="
```

Comment:
```diff
 PROVIDES_python="$PROVIDES_python
 	cmd:speedtest_cli_python
 	"
-REPLACES_python="
-	python_speedtest_cli
-	"
 PROVIDES_python3="$PROVIDES_python3
 	cmd:speedtest_cli_python3
 	"
-REPLACES_python3="
-	python3_speedtest_cli
-	"
```
I think we don't need any `REPLACES*` here, but maybe I'm missing something.

BTW, I tested the program and it works like a charm!

Reply:
Oh, I didn’t understand that at first and all the other Python packages I saw had it so I just added it just in case. It just occurred to me now that the ones listed there are probably an old naming scheme...